src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -105,7 +105,7 @@ public function getConfigTreeBuilder()
                         ->prototype('scalar')->end()
                     ->end()
                     ->arrayNode('blocked_file_types')
-                        ->prototype('scalar')->end()->defaultValue(['file/exe'])
+                        ->prototype('scalar')->end()->defaultValue([])
```

Comment:
This was never validated so from BC I will remove the default value with file/exe as I know there exist sulu projects where they have downloadable executable which I don't want to break when they update.